mcsema/BC/Instruction.cpp

```diff
@@ -212,6 +257,28 @@ llvm::Value *InstructionLifter::LiftAddressOperand(
         inst, block, arg, op);
   }
 
+  // Check if the instruction is referring to stack variable
+  if (ctx.cfg_inst->stack_var) {
+    llvm::IRBuilder<> ir(block);
+    auto map_it = ctx.cfg_inst->stack_var->refs.find(ctx.cfg_inst->ea);
+    auto base = ir.CreatePtrToInt(ctx.cfg_inst->stack_var->llvm_var, word_type);
+    auto var_offset = llvm::ConstantInt::get(word_type, static_cast<uint64_t>(map_it->second), true);
+    base = ir.CreateAdd(base, var_offset);
+    LOG(INFO) << "Lifting stack variable access at : " << std::hex << map_it->first
+        << " var_offset " << map_it->second << " variable name " << ctx.cfg_inst->stack_var->name <<std::endl;
```

Comment:
You don't need to end the `LOG` or `CHECK` macros with `std::endl`, it happens automatically. It's good to reset `std::dec` if you bring in `std::hex` because otherwise log messages start reporting line numbers in hex :-P